### ISSUE TITLE
Header of /public: Text not matching when server is configured to show local timeline only

### ISSUE THREAD

**Issue by jacotec** [tootsuite/mastodon#11374]
The public timeline of a server always shows this header text:

![466CC741-1F8A-4727-825E-EE02BBB0184C](https://user-images.githubusercontent.com/3321655/61597062-a38c2800-ac0b-11e9-82ce-239713eec31d.jpeg)

Apart from the fact that I'd love to have this configurable 😬, the text is literally wrong if the server is configured to show only the local toots here.

It should say something like this in this case:

```
Finde heraus, was hier los ist
Stöbere durch öffentliche Beiträge auf diesem Server
```

**Comment by ykzts** [comment]
I think this issue should be solved.

**Comment by jacotec** [comment]
Negative, it's NOT fixed in 3.01
